app/decorators/module_overview_decorator.rb

```diff
@@ -13,18 +13,15 @@ def call_to_action
     end
   end
 
-  # @return [Array<String, Symbol, Array>]
-  def topics_by_submodule
-    mod.items_by_submodule.map do |num, items|
-      intro = items.first
-      content = items[1..]
-      [
-        num,                                  # submodule digit
-        intro.model.heading,                  # submodule intro heading
-        previous_submodule_heading_text(num), # previous submodule heading
-        status(content),                      # symbol - all content pages in the submodule
-        topics(submodule: num, items: items), # Array(String, Symbol)
-      ]
+  # @return [Array<Hash{Symbol => String, Symbol, Array}>]
```

Comment:
I think the yard convention for multiple types is to say `Mixed`. If we are adding `Boolean` too I'd go for that option.

Reply:
I updated it as it's not an array. I've referred to here https://yardoc.org/types.html which seems to say this is right. Happy to change if you'd like though

Reply:
I just meant that `@return [Array<Hash{Symbol => Mixed}>]` is a way I've seen used to say the values of keys in the hashes can be of numerous types. One can then refer to other comments if needed. I'd be happy with that if you'd rather because the alternative is to be accurate. `position` is an `Integer`, and if another suggestion of mine was added, we'd have `Boolean` too.